kanidm_unix_int/Cargo.toml

```diff
@@ -11,37 +11,51 @@ license.workspace = true
 homepage.workspace = true
 repository.workspace = true
 
-[lib]
-name = "kanidm_unix_common"
-path = "src/lib.rs"
+[features]
+unix = []
 
 [[bin]]
 name = "kanidm_unixd"
 path = "src/daemon.rs"
+required-features = ["unix"]
```

Comment:
Wait, this gates the target based on a feature?!

Reply:
yep, but only for bins not libs 😡

Reply:
What is a bin but a lib out of hand

Reply:
Truly a wise man, this William guy!